pkg/mqtrigger/messageQueue/kafka/consumer.go

```diff
@@ -95,6 +120,7 @@ func (ch MqtConsumerGroupHandler) Cleanup(session sarama.ConsumerGroupSession) e
 		zap.Int32("generationID", session.GenerationID()),
 		zap.String("claims", fmt.Sprintf("%v", session.Claims())),
 	).Info("consumer group session cleanup")
+	ch.context.Done()
```

Comment:
We should call context cancel function from here. 
`context. Done` is receiver doesn't specify anything here.

Reply:
done